KLSadd_insertion/README.md

```diff
@@ -13,7 +13,97 @@ NOTE: The KLSadd.tex file only deals with chapters 9 and 14, as stated in the do
 
```

Comment:
Capitalize "python;" change "The updateChapters.py program" to just "updateChapters.py;" change "pdf" to ".PDF."